tools/docker-compose/bootstrap_development.sh

```diff
@@ -14,6 +14,10 @@ make awx-link
 make version_file
 
 if [[ -n "$RUN_MIGRATIONS" ]]; then
+    # wait for postgres to be ready
+    while ! nc -z postgres 5432; do
```

Comment:
Two thoughts, both probably not blocking:

1. Would it be best to have some kind of limit here? If it fails after 10-20 (or whatever) attempts, give up?
2. I did something very similar to this in another project, and ran into issues where on some ARM containers, using `nc` for this would segfault. I ended up using a small script in another language to do the test. This was with an old CentOS image though, so maybe not relevant anymore. Just something I thought of, especially as M1 and M2 become more popular.

I will approve because I don't think either of these are worth blocking on.

Reply:
thanks for taking a look. I'm content with the while loop for now, and if it causes issues later we can change to a for loop as you suggest. I'm trying not to bloat this file too much